examples/basic/basic.ipynb

```diff
@@ -469,15 +470,41 @@
     "        f\"total_transactions\",\n",
     "    ],\n",
     "    entity_rows=[\n",
-    "        GetOnlineFeaturesRequest.EntityRow(\n",
-    "            fields={\n",
-    "                \"customer_id\": Value(\n",
-    "                    int64_val=1001)\n",
-    "            }\n",
-    "        )\n",
+    "        {\n",
+    "            \"customer_id\": Value(int64_val=1001)\n",
```

Comment:
I dont want to recommend the use of this Proto API if we have a Python native API